xchange-bybit/src/main/java/org/knowm/xchange/bybit/BybitAdapters.java

```diff
@@ -53,8 +55,8 @@ public static Wallet adaptBybitBalances(List<BybitCoinWalletBalance> coinWalletB
       balances.add(
           new Balance(
               new Currency(bybitCoinBalance.getCoin()),
-              new BigDecimal(bybitCoinBalance.getEquity()),
-              new BigDecimal(bybitCoinBalance.getAvailableToWithdraw())));
+              new BigDecimal((bybitCoinBalance.getWalletBalance().isBlank()?"0":bybitCoinBalance.getWalletBalance())),
```

Comment:
The pattern we've decided on for this project is that if the server returns a blank value, then we set the value to null and not zero. Setting to zero in code could be dangerous in the case where from whatever reason the server makes a mistake and sends a blank value. Whereas when the server sends ZERO, it means zero.